x-pack/plugins/observability/public/pages/alert_details/components/header_actions.test.tsx

```diff
@@ -55,6 +63,8 @@ const mockUseFetchRuleWithoutData = () => {
   });
 };
 
+const mockOnAlertStatusChange = () => {};
+
 describe('Header Actions', () => {
```

Comment:
Does it make sense to add a test to make sure all action variables are shown correctly, and in case the rule is deleted or the user doesn't have enough permission, they are disabled correctly?

Reply:
added tests for all menu items in https://github.com/elastic/kibana/pull/174153/commits/416b995c83d9e1113343e92649ed6a1d78bdc391